creator-node/src/dbManager.js

```diff
@@ -59,14 +59,12 @@ class DBManager {
    * Deletes all data for a cnodeUser from DB (every table, including CNodeUsers)
    *
    * @param {Object} CNodeUserLookupObj specifies either `lookupCNodeUserUUID` or `lookupWallet` properties
-   * @param {?Transaction} externalTransaction sequelize transaction object
    */
-  static async deleteAllCNodeUserDataFromDB(
-    { lookupCNodeUserUUID, lookupWallet },
-    externalTransaction = null
-  ) {
-    const transaction =
-      externalTransaction || (await models.sequelize.transaction())
+  static async deleteAllCNodeUserDataFromDB({
+    lookupCNodeUserUUID,
+    lookupWallet
+  }) {
+    const transaction = await models.sequelize.transaction()
```

Comment:
why remove ability to pass in external?

Reply:
it wasn't used anywhere so it's just code bloat imo. like coding for the future but we'll probably never need to pass in external